server/Core/Entities/Sportsman.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Core.Entities;
+
+public enum Sex
+{
+    M,
+    F
+}
```

Comment:
Pink helicopters triggered :))

Reply:
I get the joke. But as far as I know gender and sex are different things. If not, well there is still no such division as "pink helicopters age 9 to 10 above 7 tons" )